### DISCUSSION TITLE
How to setText of an auto calculated field

### DISCUSSION THREAD

**Discussion by klaurtar** [Hopding/pdf-lib#1290] [Q&A]
I am working on a project to fill out a government form. One of the text fields is an auto calculated field. It multiplies one column by a percentage and calculates itself. All of my attempts to manipulate it have failed. However when I load the page in an iframe and then click another field, it auto calculates. I am wondering is there a way to cause these fields to update programmatically?

The field all the way on the right (0.00) is the one in question:
<img width="843" alt="Screen Shot 2022-08-04 at 12 51 31 PM" src="https://user-images.githubusercontent.com/43356690/182917702-9da8d0c6-2a1b-4e60-b491-40536d1e702a.png">

**Comment by xavierreis** [comment]
Hi Klaurtar, did you get a workaround for the issue? I am facing the same challenge with calculated fields.

**Reply by klaurtar** [reply]
Hey! No I did not figure it out. However, I have not put much effort in to it yet. Maybe we could do a virtual call and put our heads together to figure it out!

**Reply by klaurtar** [reply]
@xavierreis
